diff --git a/node_modules/@rnmapbox/maps/javascript/components/AbstractSource.tsx b/node_modules/@rnmapbox/maps/javascript/components/AbstractSource.tsx
index 509009a..d68961b 100644
--- a/node_modules/@rnmapbox/maps/javascript/components/AbstractSource.tsx
+++ b/node_modules/@rnmapbox/maps/javascript/components/AbstractSource.tsx
@@ -1,20 +1,17 @@
 import React from 'react';
 import { NativeMethods } from 'react-native';
 
-class AbstractSource<
-  PropsType,
-  NativePropsType extends object,
-> extends React.PureComponent<PropsType> {
-  _nativeRef?: React.Component<NativePropsType> & Readonly<NativeMethods>;
+class AbstractSource extends React.PureComponent {
+  _nativeRef?: React.Component & Readonly<NativeMethods>;
 
-  setNativeProps(props: NativePropsType) {
+  setNativeProps(props) {
     if (this._nativeRef) {
       this._nativeRef.setNativeProps(props);
     }
   }
 
   setNativeRef: (
-    instance: React.Component<NativePropsType> & Readonly<NativeMethods>,
+    instance: React.Component & Readonly<NativeMethods>,
   ) => void = (instance) => {
     this._nativeRef = instance;
   };
diff --git a/node_modules/@rnmapbox/maps/javascript/components/PointAnnotation.tsx b/node_modules/@rnmapbox/maps/javascript/components/PointAnnotation.tsx
index 506e744..47a5113 100644
--- a/node_modules/@rnmapbox/maps/javascript/components/PointAnnotation.tsx
+++ b/node_modules/@rnmapbox/maps/javascript/components/PointAnnotation.tsx
@@ -118,7 +118,7 @@ type Props = {
  * as with PointAnnotation child views are rendered onto a bitmap
  */
 class PointAnnotation extends NativeBridgeComponent(
-  React.PureComponent<Props>,
+  React.PureComponent,
   NATIVE_MODULE_NAME,
 ) {
   static defaultProps = {
diff --git a/node_modules/@rnmapbox/maps/javascript/components/ShapeSource.tsx b/node_modules/@rnmapbox/maps/javascript/components/ShapeSource.tsx
index 2dfe0ec..647555f 100644
--- a/node_modules/@rnmapbox/maps/javascript/components/ShapeSource.tsx
+++ b/node_modules/@rnmapbox/maps/javascript/components/ShapeSource.tsx
@@ -154,7 +154,7 @@ export type Props = {
  * The shape may be an url or a GeoJSON object
  */
 export class ShapeSource extends NativeBridgeComponent(
-  AbstractSource<Props, NativeProps>,
+  AbstractSource,
   NATIVE_MODULE_NAME,
 ) {
   static NATIVE_ASSETS_KEY = 'assets';
diff --git a/node_modules/@rnmapbox/maps/lib/typescript/components/AbstractSource.d.ts b/node_modules/@rnmapbox/maps/lib/typescript/components/AbstractSource.d.ts
index 284b82c..3fa9582 100644
--- a/node_modules/@rnmapbox/maps/lib/typescript/components/AbstractSource.d.ts
+++ b/node_modules/@rnmapbox/maps/lib/typescript/components/AbstractSource.d.ts
@@ -1,6 +1,6 @@
 import React from 'react';
 import { NativeMethods } from 'react-native';
-declare class AbstractSource<PropsType, NativePropsType extends object> extends React.PureComponent<PropsType> {
+declare class AbstractSource extends React.PureComponent<PropsType> {
     _nativeRef?: React.Component<NativePropsType> & Readonly<NativeMethods>;
     setNativeProps(props: NativePropsType): void;
     setNativeRef: (instance: React.Component<NativePropsType> & Readonly<NativeMethods>) => void;
diff --git a/node_modules/@rnmapbox/maps/lib/typescript/components/ShapeSource.d.ts b/node_modules/@rnmapbox/maps/lib/typescript/components/ShapeSource.d.ts
index f94230e..5811d3c 100644
--- a/node_modules/@rnmapbox/maps/lib/typescript/components/ShapeSource.d.ts
+++ b/node_modules/@rnmapbox/maps/lib/typescript/components/ShapeSource.d.ts
@@ -149,8 +149,8 @@ declare const ShapeSource_base: {
         UNSAFE_componentWillUpdate?(nextProps: object, nextState: Readonly<{}>, nextContext: any): void;
     };
 } & {
-    new (props: Props | Readonly<Props>): AbstractSource<Props, NativeProps>;
-    new (props: Props, context: any): AbstractSource<Props, NativeProps>;
+    new (props: Props | Readonly<Props>): AbstractSource;
+    new (props: Props, context: any): AbstractSource;
     contextType?: React.Context<any> | undefined;
 };
 /**
